.remarkignore

```diff
@@ -0,0 +1 @@
+CHANGELOG.md
```

Comment:
The output of the changelog confuses remark with the format of the parentheses and brackets, eg

```md
- [patch][10d96db2](https://github.com/keystonejs/keystone-5/commit/10d96db2):
```

Adding a space would remove the confusion

```diff
- - [patch][10d96db2](https://github.com/keystonejs/keystone-5/commit/10d96db2):
+ - [patch] [10d96db2](https://github.com/keystonejs/keystone-5/commit/10d96db2):
```

Is there any tooling that generates the changelogs? If so, I'd be interested in integrating [`remark-github`](https://github.com/remarkjs/remark-github), which can auto-link references to commits

Reply:
@Noviny owns this area, we're using `build-releases` which is part of [Atlaskit](https://atlaskit.atlassian.com)